### ISSUE TITLE
Add stat content to ch8

### ISSUE THREAD

**Issue by matt-dray** [DFEAGILEDEVOPS/r_training_course#8]
**Suggestion**: add detail on functions for other statistical analyses in [chapter 8 (statistical operations) ](https://github.com/dfe-analytical-services/r-training-course/blob/master/08-statistical-operations.Rmd).

**Fix**: integrate basic material on linear regression and ANOVA (plus suggestions for anything else), inspired by Upson's [Getting started with R](http://course.machinegurning.com/), for example.

**Question**: do we need more statistcial content? The chapter says 'this is not a statistics course' but does describe significance testing and t-tests. I would argue that if we've gone that far, it's trivial to add material on regressions and ANOVA, which are pretty commonplace approaches.

**Comment by TomFranklin** [comment]
Agreed this would be something really good to do, I really like example led questions from an initial question to formalising it into a hypothesis, testing etc, critique of results
